demo/admin/src/pages/errorpage/ErrorPage.tsx

```diff
@@ -0,0 +1,78 @@
+import { styled } from "@mui/material/styles";
```

Comment:
Why don't you use MUI components for this styling? Like `<Typography />` for `Message` and `<Button />` for `RetryButton`? 🤔